src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -130,7 +130,7 @@
                                       Style="{StaticResource InCanvasScrollViewerStyle}"
                                       CanContentScroll="True"
                                       Width="250">
-                            <ContentControl Template="{StaticResource MemberControlTemplate}" />
+                            <ContentControl Template="{StaticResource InCanvasMemberControlTemplate}" />
```

Comment:
There is one difference between them. `MemberControlTemplate` uses full name with signature.
And `InCanvasMemberControlTemplate` uses user friendly name, i.e. without signature.